etc/tendrl_definitions_node_agent.yaml

```diff
@@ -83,3 +92,107 @@ object_details:
       selinux_mode:
         type: String
 
+  Package:
+    enabled: True
+    attrs:
+      name:
+        help: "Location of the rpm/deb/pypi package"
+        type: String
+      version:
+        help: "Version of the rpm/deb/pypi package"
+        type: String
+      pkg_type:
+        help: "Type of package can be rpm/deb/pip/"
+      state:
+        help: "State can installed|uninstalled"
+    atoms:
+      install:
+        enabled: true
+        inputs:
+          mandatory:
+            - Package.name
+            - Package.pkg_type
+          optional:
+            - Package.version
+        name: "Install Package"
+        post_run:
+          - tendrl.node_agent.atoms.package.validations.check_package_installed
+        run: tendrl.node_agent.atoms.package.install.Install
+        type: Create
+        uuid: "16abcfd0-aca9-4022-aa0f-5de1c5a742c7"
+  Service:
+    enabled: True
+    attrs:
+      name:
+        help: "Name of the service"
+        type: String
+      state:
+        help: "Service state can be started|stopped|restarted|reloaded"
+        type: String
+      config_path:
+        help: "configuration file path for the service eg:/etc/tendrl/tendrl.conf"
+        type: String
+      config_data:
+        help: "Configuration data for the service"
+        type: String
+    atoms:
+      configure:
```

Comment:
does configure takes care of start and stop? we might have one for status as well